programs/klend/src/utils/prices/stork.rs

```diff
@@ -10,50 +10,46 @@ use crate::LendingError;
 
 pub(super) fn get_stork_price_and_twap(
     stork_price_info: &AccountInfo,
-    stork_twap_info: &AccountInfo,
+    stork_twap_info: Option<&AccountInfo>,
 ) -> Result<TimestampedPriceWithTwap> {
-    let price_feed: TemporalNumericValueFeed = stork_sdk::temporal_numeric_value::TemporalNumericValueFeed::try_deserialize(
-        &mut &stork_price_info.data.borrow()[..]
-    ).map_err(|e| {
+    let price_feed = TemporalNumericValueFeed::try_from(stork_price_info).map_err(|e| {
         msg!("Error loading stork price feed: {:?}", e);
         error!(LendingError::PriceNotValid)
     })?;
-
     let feed_id = price_feed.id;
     let price = price_feed
         .get_latest_canonical_temporal_numeric_value_unchecked(&feed_id)
-        .ok()?;
```

Comment:
Not sure if this is possible given the current setup, but ideally we would allow for Stork to support only TWAP without needing to provide spot asset I think.

Reply:
There are hacky ways to allow for this, however the klend program seems pretty opinionated about this via this struct:

```
pub(super) struct TimestampedPriceWithTwap {
    pub price: TimestampedPrice,
    pub twap: Option<TimestampedPrice>,
}
```

this indicates to me that the spot price is the first class citizen, while the twap is a nice to have but not necessary, and in fact should not exist without the relevant spot. I may be misinterpreting here so let me know your thoughts.